cmd/sql/flow.go

```diff
@@ -24,19 +24,21 @@ import (
 var (
 	configCmd   *cobra.Command
 	generateCmd *cobra.Command
+	versionCmd  *cobra.Command
 )
 
 // All cmd names
 const (
-	flowCmdName     = "flow"
-	aboutCmdName    = "about"
-	configCmdName   = "config"
-	generateCmdName = "generate"
-	initCmdName     = "init"
-	runCmdName      = "run"
-	validateCmdName = "validate"
-	versionCmdName  = "version"
-	deployCmdName   = "deploy"
+	flowCmdName            = "flow"
+	aboutCmdName           = "about"
+	configCmdName          = "config"
+	generateCmdName        = "generate"
+	initCmdName            = "init"
+	runCmdName             = "run"
+	validateCmdName        = "validate"
+	versionCmdName         = "version"
+	deployCmdName          = "deploy"
+	versionCmdOutputPrefix = "Astro SQL CLI"
```

Comment:
Can we move that to its own `const` section?

Reply:
This is no longer needed. We're now reading the json response.